starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -64,7 +64,7 @@ public Inventory create(Inventory inventory) {
    */
   public Optional<Inventory> retrieve(String id) {
     // TODO
-    return this.mongoTemplate.find(id);
```

Comment:
While this fixes your build issue, I would still figure out how to make this work. You only need 2 lines to do it.